types/qunit/index.d.ts

```diff
@@ -692,4 +692,8 @@ interface QUnit {
 }
 
 /* QUnit */
-declare const QUnit: QUnit;
+export = QUnit;
+
+declare global {
+    const QUnit: QUnit;
+}
```

Comment:
```suggestion
declare const QUnit: QUnit;

declare module 'qunit' {
    export = QUnit;
}
```

I think if you flip this around so the existing declarations remain global, the current namespace usage will continue to work and `import * as QUnit` will also act as a namespace for the types and values the way you'd want.

Reply:
👍 great catch, can't believe I didn't think of trying to invert this 😭